frontend/src/lib/components/Projects/ProjectFilter.svelte

```diff
@@ -98,6 +100,11 @@
           <Icon icon="i-mdi-script" color="text-warning" />
           {$t('project.filter.hide_drafts')}
         </ActiveFilter>
+      {:else if filter.key === 'emptyProjects' && filter.value}
+        <ActiveFilter {filter}>
+          <Icon icon="i-mdi-script" color="text-warning" />
```

Comment:
This should be changed to the same icon and text color that you end up using on line 184.